brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Off-Canvas - Add a static link instead of post link in changelog.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/7fa36381-7bd6-4ad8-8f34-b991acc614d2?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/4guKmWOK

### Types of changes




- Improvement

### How has this been tested?

- Go to the admin dashboard click on the off-canvas menu.
- Check it is opening a static link for changelog or not.

### Checklist:
